code/modules/antagonists/clock_cult/mobs/cogscarab.dm

```diff
@@ -44,6 +44,11 @@ GLOBAL_LIST_INIT(cogscarabs, list())
 		try_warp_servant(src, T, FALSE)
 	. = ..()
 
+/mob/living/simple_animal/drone/cogscarab/force_hit_projectile(obj/item/projectile/projectile)
```

Comment:
Check if the scarab is living too.
This now causes cogscarabs to be perfect cover that never breaks.